packages/esm-patient-chart-app/src/visit/visit-form/location-selection.component.tsx

```diff
@@ -34,24 +38,29 @@ const LocationSelector = () => {
     <section data-testid="combo">
       <div className={styles.sectionTitle}>{t('visitLocation', 'Visit Location')}</div>
       <div className={`${styles.selectContainer} ${styles.sectionField}`}>
-        <Controller
-          name="visitLocation"
-          control={control}
-          render={({ field: { onBlur, onChange, value } }) => (
-            <ComboBox
-              titleText={t('selectLocation', 'Select a location')}
-              aria-label={t('selectLocation', 'Select a location')}
-              id="location"
-              invalidText="Required"
-              items={locationsToShow}
-              selectedItem={value}
-              onChange={({ selectedItem }) => onChange(selectedItem)}
-              onBlur={onBlur}
-              itemToString={(loc: Location) => loc?.display}
-              onInputChange={(loc) => handleSearch(loc)}
-            />
-          )}
-        />
+        {!viewOnlyVisitLocationField ? (
```

Comment:
Hi @brandones !
I see that the config prop was renamed but it wasn't updated in here.
I'll create a new PR for this.

Thank you for the changes and review @brandones @denniskigen !

Reply:
Oh no, I'm sorry I missed that. My bad. Thank you Vineet.